packages/@lwc/ssr-compiler/src/compile-template/transformers/element.ts

```diff
@@ -136,9 +145,20 @@ function yieldAttrOrPropDynamicValue(
     cxt: TransformerContext
 ): EsStatement[] {
     cxt.import('htmlEscape');
-    const isHtmlBooleanAttr = isBooleanAttribute(name, elementName);
     const scopedExpression = getScopedExpression(value as EsExpression, cxt);
-    return [bYieldDynamicValue(b.literal(name), scopedExpression, b.literal(isHtmlBooleanAttr))];
+    switch (name) {
+        case 'class':
+            cxt.import('normalizeClass');
```

Comment:
We really need to scope these imports inside of `.import()` (just in case component authors use a variable name that conflicts). But that can be done outside of this PR.